reduce.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable indent */
+const reduce = (arr, callback, intialValue) => {
+    if(!(intialValue)) {
+        let accumlator = arr[0];
+        for (let index = 1; index < arr.length; index++) {
+        const item = arr[index];
+        accumlator = callback(accumlator, item);
+    }
+    return accumlator;
```

Comment:
It looks like your function doesn't account for the possibility of a non-falsy initial value, and it's also returning the accumulator outside the scope of the `if` block where it's defined (though since you're not testing for the non-falsy possibility it's never giving you the error you'd likely hit). I'd recommend adding a test where you call something like `reduce([1, 2, 3], thatSameCallback, 40)` - to set an arbitrary initial value - and expect the result to be 46. Then get that to pass by handling that condition in an `else` block here.